Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -57,6 +57,28 @@ + (UILocalNotification *)UILocalNotification:(id)json
 
 @implementation RCTPushNotificationManager
 
++ (NSMutableDictionary *)formatLocalNotification:(UILocalNotification *)notification
+{
+  NSMutableDictionary *formattedLocalNotification = [NSMutableDictionary dictionary];
+  
+  if (notification.fireDate) {
+    NSDateFormatter *formatter = [[NSDateFormatter alloc] init];
+    [formatter setDateFormat:@"yyyy-MM-dd'T'HH:mm:ss.SSSZZZZZ"];
+    NSString *fireDateString = [formatter stringFromDate:notification.fireDate];
+    
+    formattedLocalNotification[@"fireDate"] = fireDateString;
+  }
+  
+  formattedLocalNotification[@"alertAction"]                = RCTNullIfNil(notification.alertAction);
+  formattedLocalNotification[@"alertBody"]                  = RCTNullIfNil(notification.alertBody);
+  formattedLocalNotification[@"applicationIconBadgeNumber"] = RCTNullIfNil(@(notification.applicationIconBadgeNumber));
```

Comment:
No need to wrap this in RCTNullIfNil, `@()` will always return a non-nil object.